src/EnergyPlus/DataSurfaces.cc

```diff
@@ -734,4 +734,26 @@ Real64 AbsBackSide(EnergyPlusData &state, int SurfNum)
     return AbsorptanceFromExteriorBackSide + AbsorptanceFromInteriorBackSide;
 }
 
+void GetAddOnOverrideSurfaceList(EnergyPlusData &state)
```

Comment:
Probably should rename to `GetVariableAbsorptanceSurfaceList`?

Reply:
right, that's better